src/plugins/cams/client.py

```
@@ -0,0 +1,236 @@
+            output_path = output_dir / f"cams_eac4_{date.strftime('%Y%m%d')}.nc"
+            
+        # Format request for CDS API
+        request = {
+            'format': 'netcdf',
+            'variable': variables,
+            'date': date.strftime('%Y-%m-%d'),
+            'time': [f'{h:02d}:00' for h in range(0, 24, 3)],  # 3-hourly: 00, 03, 06, ..., 21
+            'area': area,
+        }
+        
+        logger.info(f"Downloading CAMS reanalysis for {date.date()}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)